crates/astria-composer/src/searcher/executor.rs

```diff
@@ -164,28 +205,43 @@ impl Executor {
                             break Err(e).wrap_err("failed submitting bundle to sequencer");
                         }
                     }
+
+                    block_timer.as_mut().reset(Instant::now() + Duration::from_millis(self.block_time_ms));
+                }
+
+                Some(next_bundle) = future::ready(bundle_factory.next_finished()), if submission_fut.is_terminated() => {
+                    let bundle = next_bundle.pop();
+                    if !bundle.is_empty() {
```

Comment:
A debug level event to the effect "timer is but no transactions to send" would be  great